nixos/modules/services/web-apps/rss-bridge.nix

```diff
@@ -7,6 +7,11 @@ let
 
   whitelist = pkgs.writeText "rss-bridge_whitelist.txt"
     (concatStringsSep "\n" cfg.whitelist);
+
+  configFile = pkgs.writeText "rss-bridge_config.ini" ''
+    <?php exit; ?>
+    ${lib.generators.toINI {} cfg.config}
```

Comment:
Maybe you could make use of `pkgs.formats.ini` here? That would also let you be more specific with the format for the option.

```nix
# somewhere
format = pkgs.formats.ini { };

# here
configFile = pkgs.writeText "rss-bridge_config.ini" ''
<?php exit; ?>
${format.generate cfg.config}
'';
```